front/components/UserMenu.tsx

```diff
@@ -37,11 +40,20 @@ export function UserMenu({
         {hasBetaAccess && (
           <>
             <DropdownMenu.SectionHeader label="Beta" />
-            {owner.flags.includes("labs_transcripts") && (
+            {owner.flags.includes("labs_transcripts") ||
```

Comment:
Parentheses don't work there (I don't see the labs transcripts menu anymore on my local)
That should work better :

```{(owner.flags.includes("labs_transcripts") ||
              isDevelopmentOrDustWorkspace(owner)) && (
                <DropdownMenu.Item
                  label="Transcripts processing"
                  href={`/w/${owner.sId}/assistant/labs/transcripts`}
                  icon={BookOpenIcon}
                />
              )}```

Reply:
yup! fixing